src/apprt/gtk-ng/class/debug_warning.zig

```diff
@@ -0,0 +1,80 @@
+const std = @import("std");
+const adw = @import("adw");
+const gobject = @import("gobject");
+const gtk = @import("gtk");
+
+const build_config = @import("../../../build_config.zig");
+const adw_version = @import("../adw_version.zig");
+const gresource = @import("../build/gresource.zig");
+const Common = @import("../class.zig").Common;
+
+/// Debug warning banner. It will be based on adw.Banner if we're using Adwaita
+/// 1.3 or newer. Otherwise it will use a gtk.Label.
+pub const DebugWarning = extern struct {
+    const Self = @This();
+    parent_instance: Parent,
+    pub const Parent = if (adw_version.supportsBanner()) adw.Bin else gtk.Box;
+    pub const getGObjectType = gobject.ext.defineClass(Self, .{
+        .name = "GhosttyDebugWarning",
+        .instanceInit = &init,
+        .classInit = &Class.init,
+        .parent_class = &Class.parent,
+    });
+
+    pub const properties = struct {
+        pub const debug = struct {
```

Comment:
You actually don't need this at all! This widget can assume it's always visible, and we can set the `visible` binding in window.blp! 😄